autopilot/accounts.go

```diff
@@ -146,12 +146,11 @@ func (a *accounts) refillWorkerAccounts(ctx context.Context, w Worker) {
 				defer cancel()
 				accountID, refilled, rerr := refillWorkerAccount(rCtx, a.a, w, workerID, contract)
 				if rerr != nil {
-					if inSet || rerr.Is(errMaxDriftExceeded) {
-						// register the alert on failure if the contract is in
-						// the set or the error is errMaxDriftExceeded
+					if rerr.Is(errMaxDriftExceeded) {
```

Comment:
I would still try and tackle the refill code, instead of changing when we show the alert. In case we do want to merge this we should get rid of the `inSet` variable which is now unused (see 4 lines above).